packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -20,6 +21,18 @@ class NodeCreate(InputSchemaWithoutCamelCase):
     service_id: str | None
 
 
+BootOptions: TypeAlias = dict
+
+
+class NodePatch(InputSchemaWithoutCamelCase):
+    service_version: ServiceVersion = Field(None, alias="version")
+    label: str = Field(None)
+    inputs: InputsDict = Field(None)
+    input_nodes: list[NodeID] = Field(None, alias="inputNodes")
+    progress: float | None = Field(None, ge=0, le=100)  # NOTE: MD: deprecated?
+    boot_options: BootOptions = Field(None, alias="bootOptions")
+
```

Comment:
same here? is the linter not complaining? if not then why?
What is this ```progress``` entry? what is the use-case where that would be used? let's discuss this one and remove it if there is no other reason than for it being in the current project definition.

Reply:
@odeimaiz do you need to change this progress, or can I remove it for now?

Reply:
discussed it is needed for FIle Picker progress